addons/interact_menu/functions/fnc_removeActionFromClass.sqf

```diff
@@ -17,11 +18,35 @@
  * Public: No
  */
 
-params ["_objectType", "_typeNum", "_fullPath"];
+params ["_objectType", "_typeNum", "_fullPath", ["_inherit", false, [false]]];
 
 private _res = _fullPath call FUNC(splitPath);
 _res params ["_parentPath", "_actionName"];
 
+if (_inherit) exitWith {
+    private _children = ("_x isKindOf _objectType" configClasses (configFile >> "CfgVehicles")) apply {configName _x};
+    {
+        [_x, _typeNum, _fullPath] call FUNC(removeActionFromClass);
+    } forEach (_children arrayIntersect GVAR(inheritedClassesAll)); // only need to run for classes that have already been initialized
+
+    private _index = GVAR(inheritedActionsAll) findIf { // find same path and actionName, and check if it's a parent class, needs to be checked for all classes
+        params ["_currentType", "", "_currentParentPath", "_currentAction"];
+        [_objectType isKindOf _currentType, _currentParentPath, _currentAction select 0] isEqualTo [true, _parentPath, _actionName]
+    };
+    if (_index != -1) then {
+        (GVAR(inheritedActionsAll) select _index select 4) pushBack _objectType; // add to exclude classes
```

Comment:
If you run this function x times, I believe it will execute `pushBack _objectType` x times. I'm pushing a commit that changes it to `pushBackUnique.`